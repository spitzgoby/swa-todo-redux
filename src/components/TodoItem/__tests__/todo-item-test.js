import moment from 'moment';
import React from 'react';
import {shallow} from 'enzyme';
import TodoItem from 'components/TodoItem/component';

test('<TodoItem />', () => {
  const mockTodo = {
    completed: true,
    description: 'test todo item',
    dueBy: moment('2019-01-01T00:00').format(),
    id: 1
  };
  const todoItemWrapper = shallow(<TodoItem todo={mockTodo} />);

  expect(todoItemWrapper).toMatchSnapshot();
});
