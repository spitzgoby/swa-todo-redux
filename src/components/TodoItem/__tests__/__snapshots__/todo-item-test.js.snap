// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TodoItem /> 1`] = `
<WithStyles(ForwardRef(TableRow))>
  <WithStyles(ForwardRef(TableCell))>
    <WithStyles(ForwardRef(Checkbox))
      checked={true}
      onChange={[Function]}
    />
  </WithStyles(ForwardRef(TableCell))>
  <WithStyles(ForwardRef(TableCell))>
    test todo item
  </WithStyles(ForwardRef(TableCell))>
  <WithStyles(ForwardRef(TableCell))>
    Jan 01 12:00
  </WithStyles(ForwardRef(TableCell))>
  <WithStyles(ForwardRef(TableCell))>
    <ForwardRef(Fade)
      in={false}
    >
      <WithStyles(ForwardRef(IconButton))
        color="secondary"
        onClick={[Function]}
      >
        <DeleteIcon />
      </WithStyles(ForwardRef(IconButton))>
    </ForwardRef(Fade)>
  </WithStyles(ForwardRef(TableCell))>
</WithStyles(ForwardRef(TableRow))>
`;
